R/epidist.R

```diff
@@ -15,6 +15,10 @@
 #' @param prior One or more `brmsprior` objects created by [brms::set_prior()]
 #' or related functions. These priors are passed to [epidist_prior()] in the
 #' `prior` argument.
+#' @param merge_priors If `TRUE` then merge new priors with existing ones, if
```

Comment:
Suggest to talk about the priors provided by the user rather than "new priors" and to be clearer on what "existing ones" are. As in maybe they don't know that we try to set our own defaults.